maro/simulator/core.py

```diff
@@ -175,68 +176,68 @@ def _init_business_engine(self):
         """Initialize business engine object.
 
         NOTE:
-        1. For built-in scenarios will always under "maro/simulator/scenarios" folder.
+        1. For built-in scenarios, they will always under "maro/simulator/scenarios" folder.
         2. For external scenarios, we access the business engine class to create instance.
```

Comment:
For external scenarios, we build the business engine instance with the loaded business engine class.